### DISCUSSION TITLE
窗口透明化工具

### DISCUSSION THREAD

**Discussion by QHY8IV** [share121/inter-knot#4872] [科技]
一个窗口透明化工具，可以更方便的摸鱼/办公
made by： **@qinyubo114**
[链接](https://github.com/QHY8IV/window-transparent-tool)

**Comment by obuyniq** [comment]
不是，这种码风奇烂、毫无可读性的东西就别拿来丢人现眼了吧

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >
